File: QubitServer/src/main/java/org/labrad/qubits/mem/EndSequenceCommand.java
package org.labrad.qubits.mem;

import org.labrad.qubits.FpgaModelDac;

public class EndSequenceCommand implements MemoryCommand {
  private EndSequenceCommand() {}

  private static final EndSequenceCommand INSTANCE = new EndSequenceCommand();

  public static EndSequenceCommand getInstance() {
    return INSTANCE;
  }

  public long[] getBits() {
    return new long[] {0xF00000};
  }
  public double getTime_us(FpgaModelDac dac) {
    return FpgaModelDac.clocksToMicroseconds(1);
  }
}

File: QubitServer/src/main/scala/org/labrad/qubits/mem/EndSequenceCommand.scala
package org.labrad.qubits.mem

import org.labrad.qubits.FpgaModelDac

object EndSequenceCommand extends MemoryCommand {
  def getBits(): Array[Long] = {
    Array(0xF00000)
  }

  def getTime_us(dac: FpgaModelDac): Double = {
    FpgaModelDac.clocksToMicroseconds(1)
  }
}
